CMakeLists.txt

```diff
@@ -2,10 +2,14 @@ cmake_minimum_required(VERSION 3.25)
 project(onnxruntime_extensions LANGUAGES C CXX)
 
 # set(CMAKE_VERBOSE_MAKEFILE ON)
-if(NOT CMAKE_BUILD_TYPE)
+if(NOT WIN32 AND NOT CMAKE_BUILD_TYPE)
```

Comment:
what do you expected on Win32?

Reply:
On WIN32, the config is usually set, and there can be multiple configs.

Reply:
see e.g. https://github.com/microsoft/onnxruntime-extensions/blob/1f0c76cefaa9359b9fc74ff4b9ed93214486205a/build.bat#LL23C56-L23C56

Reply:
This is right. CMake's VS generator support multiple configs. There is no need to set one when running cmake. 
See:
https://cmake.org/cmake/help/latest/prop_gbl/GENERATOR_IS_MULTI_CONFIG.html
 
Only these generators support multiple configs